go.mod

```diff
@@ -3,22 +3,26 @@ module github.com/k8snetworkplumbingwg/sriov-cni
 go 1.19
 
 require (
-	github.com/containernetworking/cni v0.8.1
-	github.com/containernetworking/plugins v0.8.7
-	github.com/onsi/ginkgo v1.2.1-0.20151202141238-7f8ab55aaf3b
-	github.com/onsi/gomega v0.0.0-20151007035656-2152b45fa28a
-	github.com/stretchr/testify v1.4.0
-	github.com/vishvananda/netlink v1.0.1-0.20190924205540-07ace697bea4
+	github.com/containernetworking/cni v1.1.2
+	github.com/containernetworking/plugins v1.1.1
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.17.0
+	github.com/stretchr/testify v1.5.1
+	github.com/vishvananda/netlink v1.2.0-beta
```

Comment:
is netlink version bump needed ? if so do you know what was added/fixed ?

Reply:
Yes, In the latest release of `netlink`,  they have added the [RouteAddEcmp](https://github.com/vishvananda/netlink/blob/337a587de15022938a9a31d3f9776faeb5c9f8d7/route_linux.go#L678) function, which is the most important part.

Reply:
OK here is what i found:

support for calling `RouteAddEcmp` was added to `containernetworking/plugins` in v1.0.0 in [this commit](https://github.com/containernetworking/plugins/commit/76ef07ebc6dcfd4fa7c71720544449d6f8e6d693)

in sriov-cni code [we call](https://github.com/k8snetworkplumbingwg/sriov-cni/blob/301a1ffce767a446beb7881402ef1bea99bad71b/cmd/sriov/main.go#L138) `ConfigureInterface` from containernetworking plugins ipam package.


so we need to bump containernetworking/plugins verison.

in v1.1.1 an older version of netlink[ is used](https://github.com/containernetworking/plugins/blob/4744ec27b89c083194e7df498de50f03a8a1d3ec/go.mod#L21)
which apparently is good enough, as bumping netlink version here will not affect the netlink version set in plugins go.mod  (we are using go modules to build so each package provides its own dependencies)

still, i do not think netlink version should be bumped as there is no reason to bump it.
am i missing anything ?

Reply:
ah i see latest PR update now uses netlink v1.1.1 and not 1.2.0